students/10/main.js

```diff
@@ -23,10 +23,41 @@ const ex4 = () => {
     console.log(countLetters(array)); 
 }
 
+const ex6 = () => {
+    let array = ["this", "is", 1, 3, 2.1, "a", "test"];
+    let result = countItems(array);
+    console.log("Number of Integers: " + result.numIntegers)
+    console.log("Number of Decimal Numbers: " + result.numDecimals)
+    console.log("Number of Strings: " + result.numStrings)
+}
+
 //
 // Your functions here...
 //
 
+const countItems = (array) => {
+    let countDec = 0;
+    let countStr = 0;
+    let countNum = 0;
+    for(let i = 0; i < array.length; i++) {
```

Comment:
The assignment states the following:

This arrow function calls the following arrow functions:

countStrings
countIntegerNumbers
countDecimalNumbers

Reply:
update to arrow functions

Reply:
Well, I'm really looking for this sort of structure/architecture:

```
const countItems = () => {

      let numStrings = countStrings(array);
      let numIntegers = countIntegerNumbers(array);
      let numDecimalNumbers = countDecimalNumbers(array);
}
````

In this approach, the function `countItems` calls other functions to help get the job done.